Plugins/OutText.Internal.ps1

```diff
@@ -161,6 +181,18 @@
                 ## Fix Set-StrictMode
                 if (Test-Path -Path Variable:\Options) { $options = Get-Variable -Name Options -ValueOnly; }
                 else { $options = New-PScriboTextOptions; }
+
+                try{
```

Comment:
@wiltaylor It doesn't look like the `OutTextLineBreak` changes have been fully merged? It looks like you've added the `TextWidth` check, but not updated/moved the others?